cJSON.h

```diff
@@ -117,9 +117,10 @@ typedef struct cJSON
     int valueint;
     /* The item's number, if type==cJSON_Number */
     double valuedouble;
-
+	uint64_t  valueint64;
```

Comment:
change the cJSON struct and add new feature of C99 is a break change, it will cause incompatibility with C89. I think it should be implemented in v2.0

Reply:
crap... the int64 is a temp feature i added since i need int64 support and i think it's implemented badly as  i need it now and didn't have time to architect it properly.
i'll open a new PR without it, maybe from a side branch.
I was sure the PR will not change after a commit to the tip of the master :-\